SimG4Core/PhysicsLists/src/CMSMonopolePhysics.cc

```diff
@@ -112,9 +112,9 @@ void CMSMonopolePhysics::ConstructProcess() {
       }
   
       if (magn != 0.0) {
-	pmanager->RemoveProcess(0);
-	pmanager->AddProcess(new MonopoleTransportation(mpl,chordFinderSetter,verbose),
-                                                        -1, 0, 0);
+        G4int idxt(0);
```

Comment:
@fabiocos , this may be false positive warning. The problem is in two methods of Genat4 class RevoveProcess(..), in first the argument int, in another G4VProcess*. If the value is 0, compiler may be confused which one to choose.

Reply:
@civanch ok, thank you for the clarification, I see the reason